### DISCUSSION TITLE
How to increase privileges control on data level

### DISCUSSION THREAD

**Discussion by timgogochen** [Budibase/budibase#6599] [Help] [answered]
Dear Developer

Very thanks for this amazing product , and it easy and power full to use to prototype realization .
Under current days , we trying to build on requirement management system  , we need to limited only requester can be access him records when same APP shared to many users, and users role on basic level, we can using condition function to limit if userid not equa session users id the form will be hide , but we cannot limit api and restful data exposure， would your proive any advise?

Thanks.

**Comment by mike12345567** [comment, marked as answer]
Hi @timgogochen currently this is not a feature that is supported by Budibase, you can track its development on the following issue: https://github.com/Budibase/budibase/issues/4789

**Comment by timgogochen** [comment]
Hi Mike

Thanks for your reply , actually the authorization control is very basic and useful features in real transaction system , so if development team do not have a schedule to improvement , would you please provide some solution to guide us to utilize on productive system. just learning.

Thanks and B.R
